### PR TITLE
chore: upgrade Electron to `22.3.0`

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

To help with the Snyk CI failures, we need to update Electron to at least `22.2.0`.

## Changelog

```
- Increment Electron version to 23.1.0
- Update yarn.lock file
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [x] MacOS
    -   [x] Linux
    -   [x] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

Test general app functionality.

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by Tuditi — APPROVED** [review]
Works well, but merging this means we won't support windows 7/8/8.1.

File: packages/desktop/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
         "copy-webpack-plugin": "^7.0.0",
         "cross-env": "^7.0.2",
         "css-loader": "^5.0.1",
-        "electron": "22.0.0",
+        "electron": "23.1.0",
```
Comment (Tuditi) [inline packages/desktop/package.json:46]: Let's upgrade it to the latest 22 version, because otherwise we'll have windows support issues.

**Review by Tuditi — CHANGES_REQUESTED** [review]

File: packages/desktop/package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
         "copy-webpack-plugin": "^7.0.0",
         "cross-env": "^7.0.2",
         "css-loader": "^5.0.1",
-        "electron": "22.0.0",
+        "electron": "22.2.1",
```
Comment (Tuditi) [inline packages/desktop/package.json:46]:
```suggestion
        "electron": "22.3.0",
```

**Review by Tuditi — CHANGES_REQUESTED** [review]
There has been a newer electron release with a security fix.

**Review by nicole-obrien — APPROVED** [review]
Works on Mac

**Pull request merged**